src/components/GGanttChart.vue

```diff
@@ -153,6 +184,32 @@ const getChartRows = () => {
   return allBars
 }
 
+const labels = computed(() => {
```

Comment:
This is basically a duplicate of  `getChartRows()`. It'd be better to extend  `getChartRows()` so that it collects the row labels alongside the bars.